bootstrap/src/errors.py

```diff
@@ -53,16 +53,16 @@ def __init__(self, extra_param):
 class DestinationDoesNotExistError(BootstrapError):
     def __init__(self, destination):
         self.code = 4002
-        self.http_code = 400
+        self.http_code = 404
         message = f"Destination '{destination}' does not exist"
         super(DestinationDoesNotExistError, self).__init__(message)
 
 
 class LowBalanceError(BootstrapError):
     def __init__(self):
         self.code = 4003
-        self.http_code = 400
-        message = f'The account does not have enough kin to perform this operation'
+        self.http_code = 500
```

Comment:
why was this changed? this doesn't match the api.yaml

Reply:
I was originally debating making this a 500, since 4xx series codes imply to the client that there is something they can change about the request and retry with expected success and a low balance isn't really a client "mistake".

I ended up leaving it as is in `api.yaml` since you could reasonably say that the client can simply send less kin, and succeed on the request, and that also minimized externally facing changes.

I've pushed a commit (d9eca56bf25864bc58f2dff8ed48fe0d0849c99f) to the PR to re-align the python with the yaml.